### PR TITLE
Update build.md

### PR DESCRIPTION
**Pull request by faddat**
added LLVM to the dependency one-liner and dependency list for 

https://github.com/saulius/croaring-rs

### CODE REVIEW

File: doc/build.md [hunk 1]
```diff
@@ -119,4 +120,4 @@ troubleshooting, etc.
 
 Please note that all mining functions for Grin have moved into a separate, standalone package called
 [grin_miner](https://github.com/mimblewimble/grin-miner). Once your Grin code node is up and running,
-you can start mining by building and running grin-miner against your running Grin node.
\ No newline at end of file
+you can start mining by building and running grin-miner against your running Grin node.
```
Comment (RyanZim) [inline doc/build.md]: What's the change here?

Comment (faddat) [reply doc/build.md]: There seems to be none!  One moment while I fix this PR.

Comment (faddat) [reply doc/build.md]: Oops.  The change is there.  

https://github.com/mimblewimble/grin/pull/1975/files

I just added LLVM to the list of dependencies and to the dependency one-liner.

Comment (faddat) [reply doc/build.md]: No idea what the change there is.

Comment (faddat) [reply doc/build.md]: When I check here:  

https://github.com/mimblewimble/grin/pull/1975/files?short_path=10ebff8#diff-10ebff85ac4d31e01c65612e886ae89a

It only shows the llvm changes.

**Comment by sesam** [comment]
@ jakob The surprise change is a new line (enter) that was added to the
list line by your text editor, as is best practice. So all is good.
lgtm - This is ready be merged.

**Pull request merged**